gapic/src/main/com/google/gapid/models/Settings.java

```diff
@@ -350,6 +343,7 @@ private static boolean getBoolean(Properties properties, String name, boolean df
     }
   }
 
+  /*
```

Comment:
Commented code? Can it be removed?

Reply:
Yeah, When commenting it out I thought it was worth keeping, but that doesn't really make sense, since it's in the history...